code/modules/power/singularity/emitter.dm

```diff
@@ -195,6 +195,10 @@
 	if(. && !anchored)
 		step(src, get_dir(user, src))
 
+/obj/machinery/power/emitter/attack_ai_secondary(mob/user, list/modifiers)
```

Comment:
Looks like this is only the AI. Are you sure this works with cyborgs?

Reply:
Yeah borgs call this proc unless theirs is overridden:
```
/atom/proc/attack_robot_secondary(mob/user, list/modifiers)
	return attack_ai_secondary(user, modifiers)
```

Reply:
It's not that I don't think you didn't test it, but it's that it's very common for edge cases to not come up in personal playtesting.

Reply:
Just want to clear up all doubts. No offense intended power mover blocks.